verification/src/http_server/handlers/verification/mod.rs

```diff
@@ -27,9 +21,10 @@ pub struct VerificationResult {
     pub compiler_version: String,
     pub evm_version: String,
     pub constructor_arguments: Option<String>,
-    pub contract_libraries: Option<Vec<ContractLibrary>>,
+    pub optimization_runs: Option<usize>,
+    pub contract_libraries: BTreeMap<String, String>,
     pub abi: String,
-    pub sources: HashMap<String, String>,
+    pub sources: BTreeMap<String, String>,
```

Comment:
to sort the sources in alphabetical order? is it important?

Reply:
I saw that `ethers_solc` uses BTreeMap for `sources`, and you used it for `libraries` field. For me it's just convenient to use same structures for same fields
<img width="441" alt="изображение" src="https://user-images.githubusercontent.com/41516657/173142058-3ad13fe5-bb5c-4705-a8ce-10edd4df82df.png">

Reply:
then why not just use ether_solc::Sources?

Reply:
In my opinion, direct mention of BTreeMap is better than type alias. Also `Sources` have `PathBuf` as key, and it will create unnecessary type conversions with error handling

Reply:
not sure about pathbuf being a bad thing, but I missed that it's just a type alias

> direct mention of BTreeMap is better than type alias

yes and no. It is easier to read, yes, but imagine the library will change the type, for example from BTreeMap to HashMap, then you'll need to fix it in your code, while if you're using `Sources` the code might work fine 

but it's just food for thought, not asking to replace it (only if you want :) )